### PR TITLE
url params for Competitions/Submissions

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1656

### CODE REVIEW

File: src/lib/proposalUtils.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { copyToClipboard } from "./util";
+import moment = require("moment-timezone");
```
Comment (orenyodfat) [inline src/lib/proposalUtils.ts:2]: how this is related to this pr ?

File: src/lib/proposalUtils.ts [hunk 2]
```diff
@@ -23,7 +24,11 @@ export function importUrlValues<Values>(defaultValues: Values) {
           initialFormValues[prop] = BigInt(paramValue);
           break;
         case "object":
-          initialFormValues[prop] = JSON.parse(paramValue);
+          if (moment.isMoment(defaultValues[prop])) {
+            initialFormValues[prop] = moment(paramValue);
+          } else {
+            initialFormValues[prop] = JSON.parse(paramValue);
```
Comment (orenyodfat) [inline src/lib/proposalUtils.ts:30]: how this is related to this pr ?

File: src/lib/proposalUtils.ts (hunk 1 above)
Comment (dkent600) [reply src/lib/proposalUtils.ts:2]: Competition parameters include dates.  This is to support them.

File: src/lib/proposalUtils.ts (hunk 2 above)
Comment (dkent600) [reply src/lib/proposalUtils.ts:30]: Competition parameters include dates.  This is to support them.

**Pull request merged**